### PR TITLE
finish distributed sytem, and test word counting of two files

### PR DESCRIPTION
**Pull request by survivingME**
finish data computing platform based on alluxio(memory) and distributed computing model
Test word count of two files, the output as:
10.2.152.21:7778 : Try to call Master.Register: 
10.2.152.21:7778 : Call Master.Register done.
10.2.152.22:7778 : Try to call Master.Register: 
10.2.152.22:7778 : Call Master.Register done.
creating mrtmp.wcseq- 1 - 0
creating mrtmp.wcseq- 1 - 1
10.2.152.21:7778: Map task #1 done
creating mrtmp.wcseq- 0 - 0
creating mrtmp.wcseq- 0 - 1
10.2.152.22:7778: Map task #0 done
10.2.152.21:7778: Reduce task #1 done
10.2.152.22:7778: Reduce task #0 done

### CODE REVIEW

File: src/main/wc.go [hunk 1]
```diff
@@ -55,17 +56,24 @@ func reduceF(key string, values []string) string {
...
     if len(os.Args) < 4 {
         fmt.Printf("%s: see usage comments in file\n", os.Args[0])
     } else if os.Args[1] == "master" {
-        var mr *mapreduce.Master
+        /*var mr *mapreduce.Master
         if os.Args[2] == "sequential" {
             mr = mapreduce.Sequential("wcseq", os.Args[3:], 3, mapF, reduceF)
         } else {
-            mr = mapreduce.Distributed("wcseq", os.Args[3:], 3, os.Args[2])
+            mr = mapreduce.distributed("wcseq", os.Args[3:], 3, os.Args[2])
```
Comment (bupt-wenxiaole) [inline src/main/wc.go]: 大写？

File: src/mapreduce/test_test.go [hunk 2]
```diff
@@ -125,48 +157,52 @@ func port(suffix string) string {
...
 func TestBasic(t *testing.T) {
 	mr := setup()
-	for i := 0; i < 2; i++ {
+	/*for i := 0; i < 2; i++ {
 		//go RunWorker(mr.address, port("worker"+strconv.Itoa(i)),
 		go RunWorker(mr.address, port(":"+strconv.Itoa(1500+i)),
 			MapFunc, ReduceFunc, -1)
-	}
-	mr.Wait()
+	}*/
```
Comment (bupt-wenxiaole) [inline src/mapreduce/test_test.go:199]: let this part work

File: src/mapreduce/test_test.go [hunk 3]
```diff
@@ -125,48 +157,52 @@ func port(suffix string) string {
...
+	//let three workers run
+	go RunWorker("10.2.152.21", port(":"+strconv.Itoa(1500)), MapFunc, ReduceFunc, -1)
+	go RunWorker("10.2.152.22", port(":"+strconv.Itoa(1500)), MapFunc, ReduceFunc, -1)
+	go RunWorker("10.2.152.24", port(":"+strconv.Itoa(1500)), MapFunc, ReduceFunc, -1)
+	mr.wait()
 	check(t, mr.files)
 	checkWorker(t, mr.stats)
 	cleanup(mr)
```
Comment (bupt-wenxiaole) [inline src/mapreduce/test_test.go]: plz format code

**Review by bupt-wenxiaole — COMMENTED** [review]
OK，plz modify it

**Comment by survivingME** [comment]
delete dependent files, modify somewhere is not in format and some of  .sh, but not finish TestBasic

**Pull request merged**